.travis.yml

```diff
@@ -1,27 +1,98 @@
 language: rust
+sudo: false
 branches:
   only:
     - master
-os:
-  - linux
-  - windows
-rust:
-  - stable
-  - beta
-  - nightly
+
 # Need to cache the whole `.cargo` directory to keep .crates.toml for
 # cargo-update to work, but don't cache the cargo registry
 before_cache:
   - rm -rf /home/travis/.cargo/registry
 cache:
   directories:
     - /home/travis/.cargo
-before_script:
-  - if [[ "$TRAVIS_RUST_VERSION" == "stable" || "$TRAVIS_RUST_VERSION" == "nightly" ]]; then rustup component add clippy-preview; fi
-script:
-  - if [[ "$TRAVIS_RUST_VERSION" == "stable" || "$TRAVIS_RUST_VERSION" == "nightly" ]]; then cargo clippy --all-targets --all-features -- -D warnings; fi
-  - cargo test --verbose
+
+DEPLOY_TO_GITHUB: &DEPLOY_TO_GITHUB
+  before_deploy:
+    - git config --local user.name "Stuart Crust"
+    - git config --local user.email "stuart.crust@gmail.com"
+    - name="supernova-$TRAVIS_TAG-$TARGET"
+    - mkdir $name
+    - cp target/$TARGET/release/supernova $name/
+    - cp README.md LICENSE-MIT LICENSE-APACHE supernova.sha1 $name/
+    - tar czvf $name.tar.gz $name
+  deploy:
+    provider: releases
+    api_key: $GH_TOKEN
```

Comment:
What happens if `GH_TOKEN` isn't defined in the environment?

Reply:
I had this happen but can't find the build log - Travis threw an error. Will recreate and message here

Reply:
Here's the result of not having `GH_TOKEN` defined: https://travis-ci.org/SeanPrashad/supernova/jobs/460614865#L1016

![image](https://user-images.githubusercontent.com/13009507/49132057-a024e480-f2a8-11e8-83ed-efd8a821770f.png)